store/src/main/java/com/dropbox/android/external/store4/impl/RealStore.kt

```diff
@@ -83,10 +84,11 @@ internal class RealStore<Key : Any, Input : Any, Output : Any>(
             @Suppress("UNCHECKED_CAST")
             createNetworkFlow(
                 request = request,
+                sourceOfTruthCacheType = null,
                 networkLock = null
             ) as Flow<StoreResponse<Output>> // when no source of truth Input == Output
         } else {
-            diskNetworkCombined(request, sourceOfTruth)
+            sourceOfTruthNetworkCombined(request, sourceOfTruth)
         }.onEach {
             // whenever a value is dispatched, save it to the memory cache
             if (it.origin != ResponseOrigin.Cache) {
```

Comment:
this is probably broken as it will write the value into cache multiple times. (haven't verified yet, just putting here not to forget)